reference/p2p/getgraphene.rst

```diff
@@ -0,0 +1,22 @@
+.. Copyright (c) 2019 The Unit-e developers
+   Distributed under the MIT software license, see the accompanying
+   file LICENSE or https://opensource.org/licenses/MIT.
+
+getgraphene
+-----------
+
+Contains a p2p::GrapheneBlockRequest Sent in response "headers" message
```

Comment:
in response `to`?

Reply:
Sounds better, yes. That needs to be fixed in https://github.com/dtr-org/unit-e/blob/3162a6f63b09b26d630d6eca1baadbe7765a4c42/src/protocol.h#L277